tests/test_transport.py

```diff
@@ -116,7 +116,7 @@ def test_simple_rate_limits(httpserver, capsys, caplog, make_client):
     client.capture_event({"type": "transaction"})
     client.flush()
 
-    assert len(httpserver.requests) == 1
+    assert list(r.url[-18:] for r in httpserver.requests) == ["/api/132/envelope/"]
```

Comment:
`url[-18:]` :-(

I'd keep the original check for 1 request, then you can `assert httpserver.requests[0].endswith("/api/132/envelope/"]`

Reply:
yeah fair